services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from db.models import MovieSession, Movie, CinemaHall
+from datetime import time
+from django.db.models import QuerySet
+
+
+def create_movie_session(movie_show_time: time,
+                         cinema_hall_id: int,
+                         movie_id: int,) -> MovieSession:
+
+    session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+    return session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    if movie_session_id:
+        return MovieSession.objects.get(id=movie_session_id)
+    else:
+        return "No such Id"
```

Comment:
I'm not sure, that `movie_session_id` can be empty
```suggestion
    return MovieSession.objects.get(id=movie_session_id)
```